guides/Visual_design/color theory/color theory.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+Color theory is a combination of science and art of color. It explains how humans perceive color, how colors mix, match or clash and various ways in which colors can be classified and used.
+
+# RGB COlOR MODEL
+
+This model uses the primary colors red green and blue in different proportions to create all the necessary colors .When you mix red,green and blue colors to their full extent i.e 100% the resultant color is white and when they are mixed to their lowest degree the resultant color is black. So basically you start with black and as you add lights of various colors you get white. Thus RGB color model is also called as additive color model. Colors are much more vibrant in case of rgb as compared to cmyk.
```

Comment:
1) Insert a comma between, red and green. 
2) Extra space between colors and full stop.